### PR TITLE
Make rvm_builder work with IPs or node names, add Ruby 2.3.1

### PR DESCRIPTION
**Pull request by mcasper**

### CODE REVIEW

**Comment by ajvb** [comment]
Wow, bad docs, my bad. +1

File: rvm-builder/run-builder.sh [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
 
+# If the user passed in an IP, we need to use the -S flag
+if [[ "$NODE_INPUT" =~ ^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$ ]]; then
+  salt_target="-S ${NODE_INPUT}"
+# If the user passed a node name, no need to do anything special
+else
+  salt_target="${NODE_INPUT}"
+fi
+
+
```
Comment (mcasper) [inline rvm-builder/run-builder.sh]: Oops. extra whitespace

**Comment by mcasper** [comment]
No worries :)

**Pull request merged**